### DISCUSSION TITLE
Anyone running moose applications through visual studio code?

### DISCUSSION THREAD

**Discussion by adhithya96** [idaholab/moose#26823] [Q&A General]
I saw a thread where someone had mentioned about using files like launch.json, task.json but it's  not working for me.

**Comment by GiudGiud** [comment]
@loganharbour @dschwen  do you know someone who uses this

**Reply by adhithya96** [reply]
I'm new to moose. I wanted to understand how to use a debugger so that I can understand the workflow for developing application. I have found Ubuntu's gdb debugger for now which I'm using.
